js/src/Ice/ObjectAdapter.js

```diff
@@ -29,6 +29,11 @@ import { Debug } from "./Debug.js";
 const _suffixes = [
     "ACM",
```

Comment:
Should "ACM" be deleted from this list, now that it's been removed and replaced?

Reply:
Yes, for a follow-up PR.